newrelic-agent/src/main/resources/newrelic.yml

```diff
@@ -120,16 +120,9 @@ common: &default_settings
       # When true, the agent will decorate your application log files with linking metadata. The default is false.
       #enabled: false
 
-  # Adds integration with CodeStream, introducing Code-Level Metrics! Golden Signals visible in your
-  # IDE through New Relic CodeStream.
-  code_level_metrics:
-    # When true the agent will capture namespace and function information
-    # on spans to enable code level metrics in CodeStream.
-    enabled: false
-
```

Comment:
code_level_metrics and comments need to stay in. This is a new feature and new config that was recently merged and will release soon.